CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+namespace CanHazFunny;
+
+public class Jester
+{
+	public JokeService _jokeService { get; set; }
+	public JokeOutput _jokeOutput { get; set; }
```

Comment:
```suggestion
	private readonly JokeService _JokeService;
	private readonly JokeOutput _JokeOutput;
```
Do these need to be public? And make sure you are using the correct naming conventions